### ISSUE TITLE
"send API request" request sent to wrong server IP

### ISSUE THREAD

**Issue by smitabh84** [stoplightio/elements#2212]
When user clicks on "Send API request" the request should be sent to the servr IP that user has selected, currently all the requests are going to nutanix.dev. How can I handle it in an embedded setup like Nutanix? Screenshot for error and project setting is attached.

So basically the "send API request" is useless for our customers Please check it ASAP.

## Context
![image](https://user-images.githubusercontent.com/66934062/182555838-be0074d4-afc3-41e8-8815-fdef15c88509.png)

![image](https://user-images.githubusercontent.com/66934062/182555210-1ab8a70a-aee6-4cde-86de-6ab8a55ce350.png)

## Steps to Reproduce

1.
2.
3.
4.

## Environment

- Version used:
- Environment name and version (e.g. Chrome 39, node.js 5.4):
- Operating System and version (desktop or mobile):
- Link to your environment/workspace/project:

**Comment by mnaumanali94** [comment]
Hey. Do you have the server set in your OpenAPI file?

**Comment by mnaumanali94** [comment]
@smitabh84

**Comment by smitabh84** [comment]
yes. u can check the nutanix.dev for the project. I have set all the details.
